docs/manual/introduction.tex

```diff
@@ -1276,13 +1280,12 @@
 \item
   A stub file (Section~\ref{stub}) can override the library's annotations,
   for one or more methods.
+\item
+  Don't type-check clients.  JUnit's \<assertNotNull()> is typically called
```

Comment:
Should `clients` here be `tests`?   I guess I'm a little confused what `clients` means in this context.